SQL/2016-03-25-GenomicBrowser-CPG.sql

```diff
@@ -0,0 +1,64 @@
+-- ALTER TABLE genome_loc ADD Description text, MODIFY Strand enum ('forward', 'reverse');
```

Comment:
included in https://github.com/aces/Loris/pull/1572. 
This is going to be fixed when I will rebase.